cmd/crates/soroban-spec-tools/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "soroban-spec-tools"
+description = "Tools for using a contract's XDR spec"
+homepage = "https://github.com/stellar/soroban-tools"
```

Comment:
should these refs to soroban-tools be changed soroban-rpc now?

Reply:
@sreuland  Yes already on it :)